src/Core/Variables/VariableExtractor.php

```diff
@@ -184,6 +185,8 @@ protected function extractWithAccessor($subject, $propertyName, $accessor)
                 return call_user_func_array([$subject, 'get' . ucfirst($propertyName)], []);
             } elseif ($accessor === self::ACCESSOR_ASSERTER) {
                 return call_user_func_array([$subject, 'is' . ucfirst($propertyName)], []);
+            } elseif ($accessor === self::ACCESSOR_ASSERTER_HAS) {
```

Comment:
The `isExtractableThroughAsserted` method is fine - maybe we should add an `extractThroughAsserter` method and call that if `self::ACCESSOR_ASSERTER` is used. See first comment in review.